public/app/features/datasources/DataSourcesList.tsx

```diff
@@ -1,39 +1,58 @@
 // Libraries
-import React, { PureComponent } from 'react';
+import React, { FC } from 'react';
 import classNames from 'classnames';
 
-// Components
-import DataSourcesListItem from './DataSourcesListItem';
-
 // Types
 import { DataSourceSettings } from '@grafana/data';
 import { LayoutMode, LayoutModes } from '../../core/components/LayoutSelector/LayoutSelector';
+import { Card, Tag, useStyles } from '@grafana/ui';
+import { css } from 'emotion';
 
 export interface Props {
   dataSources: DataSourceSettings[];
   layoutMode: LayoutMode;
 }
 
-export class DataSourcesList extends PureComponent<Props> {
-  render() {
-    const { dataSources, layoutMode } = this.props;
-
-    const listStyle = classNames({
-      'card-section': true,
-      'card-list-layout-grid': layoutMode === LayoutModes.Grid,
-      'card-list-layout-list': layoutMode === LayoutModes.List,
-    });
+export const DataSourcesList: FC<Props> = ({ dataSources, layoutMode }) => {
+  const listStyle = classNames({
+    'card-section': true,
+    'card-list-layout-grid': layoutMode === LayoutModes.Grid,
+    'card-list-layout-list': layoutMode === LayoutModes.List,
+  });
```

Comment:
Don't think this is needed as I haven't noticed any other components using `layoutMode`, but not 100% sure it's safe to delete.

Reply:
yea, you can delete these, not used